### PR TITLE
Add place names and child place types to all relevant charts

### PR DESCRIPTION
**Pull request by pradh**
I think the test diffs give an idea of the title change.  I've sanity checked the demo scripts.

Pasting just a couple of screenshots (there are too many).

Screenshot 2023-02-06 at 6 34 23 PM
Screenshot 2023-02-06 at 6 34 37 PM

### CODE REVIEW

File: server/lib/nl/page_config_next.py [hunk 1]
```diff
@@ -264,10 +267,13 @@ def _multiple_place_bar_block(column, places: List[Place], svs: List[str],
     title = attr['title']
   elif len(svs) > 1:
     # This suggests we are comparing against SV peers from SV extension
-    title = 'Compare with Other Variables'
+    title = 'Comparison with Other Variables'
```
Comment (beets) [inline server/lib/nl/page_config_next.py:270]: How about "Compared with"? Otherwise should be "In comparison with"

**Review by beets — COMMENTED** [review]
Might I suggest a `_decorate_title` that takes in a dictionary of possible flags? That will give further flexibility with the titles, since  now they are dependent on the call order.

**Comment by pradh** [comment]
> Might I suggest a `_decorate_title` that takes in a dictionary of possible flags? That will give further flexibility with the titles, since now they are dependent on the call order.

Good idea!  I added the function.  But it could not take in an attr dictionary (because for e.g., the same attr we want with-pc and without-pc charts).

PTAL.

File: server/lib/nl/page_config_next.py (hunk 1 above)
Comment (pradh) [reply server/lib/nl/page_config_next.py]: Done.

**Comment by pradh** [comment]
Thanks for the review!

**Review by beets — APPROVED** [review]

**Pull request merged**